tests/Functional/ContentRenderTestBase.php

```diff
@@ -205,4 +206,118 @@ protected function createContactEntity(string $name, string $bundle, int $status
     return $contact;
   }
 
+  /**
+   * Asserts Contact entity field rendering.
+   *
+   * @param \Behat\Mink\Element\NodeElement $element
+   *   Rendered element.
+   * @param string $name
+   *   Name of the Contact entity.
+   */
+  protected function assertContactEntityDefaultDisplay(NodeElement $element, string $name): void {
```

Comment:
ContentProjectRenderTest and OrganisationRenderTest have methods to asser the rendering of contact items. Could we check if we can abstract this methods and have a unified way of asserting this? If so, open a follow up.